tests/Unit/Subscriber/Cache/PurgeActionsSubscriber/purgeUserCache.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace WP_Rocket\Tests\Unit\Subscriber\Cache\PurgeActionsSubscriber;
+
+use WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+use Brain\Monkey\Filters;
+
+/**
+ * @covers \WP_Rocket\Subscriber\Cache\PurgeActionsSubscriber:purge_user_cache
+ * @group purge_actions
+ */
+class Test_PurgeUserCache extends TestCase {
+	/**
+	 * Test should not purge user cache when user cache is disabled
+	 */
+	public function testShouldNotPurgeUserCacheWhenUserCacheDisabled() {
+		$options = $this->createMock( 'WP_Rocket\Admin\Options_Data' );
+		$map     = [
+			[
+				'cache_logged_user',
+				0,
+				0,
+			],
+		];
+
+		$options->method( 'get' )->will( $this->returnValueMap( $map ) );
+
+		$purge = new PurgeActionsSubscriber( $options );
+		$this->assertNull( $purge->purge_user_cache( 1 ) );
```

Comment:
`$this->assertNull()` doesn't tell us anything. Why? The method always returns `null` whether purging or not. Instead, use BrainMonkey to mock that `rocket_clean_user()` never runs.

```php
Functions\expect( 'purge_user_cache' )->never();

$purge = new PurgeActionsSubscriber( $options );
$purge->purge_user_cache( 1 );
```